web-components/src/[sandbox]/examples/combobox.ts

```diff
@@ -14,12 +19,7 @@ export const comboBoxTemplate = html`
     placeholder="PlaceholderPlaceholderPlaceholderPlaceholderPlaceholder"
   ></md-combobox>
   <h3>Default with trim space</h3>
-  <md-combobox
-    .options=${comboBoxOptions}
-    placeholder="Placeholder"
-    .value=${[comboBoxOptions[5]]}
-    search-trim-space
```

Comment:
Does `serch-trim-space` no longer do anything? I'm confused as to why it's just being removed here, no longer shown in any part of the sandbox.

Reply:
I was just do it for demonstrate Alexis how she can use "value" and "Options". But now returned all back, as we discussed it with with her and she understand how she can use those properties.